src/components/doughnutChart/DoughtnutChartComponent.tsx

```diff
@@ -0,0 +1,10 @@
+import { Doughnut } from 'react-chartjs-2';
```

Comment:
if the component is DoughtnutChart use the same name in the file name `DoughtnutChart.tsx`

as well the component label is redundant since this is already under /components forlder 

/the directory need to be in uppercase

Reply:
On previous ticket you told me to use cammelCase to name my directories and components. Should I disregard that for this current ticket?

Reply:
yeah sorry but use the same naming a the component folder